README.md

```diff
@@ -29,6 +29,15 @@ npm run serve
 TV_IP=10.0.1.1 npm run test-tv
 ```
 

```

Comment:
When it is not agate, there is no explanation as to which component is running.
How about mentioning that existing commands run on sandstone?